kodilla-testing/src/main/java/com/kodilla/testing/collection/OddNumbersExterminator.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.testing.collection;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class OddNumbersExterminator {
+
+    List<Integer> resultNumbers = new ArrayList<>();
```

Comment:
to powinna byc zmienna lokalna metody exterminate. W przeciwnym wypadku, ta lista bedzie "puchnac" przy kazdym wywolaniu